# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_reply_without_content_type.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_reply_without_content_type.js
--- a/netwerk/test/unit/test_reply_without_content_type.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_reply_without_content_type.js	Tue Feb 12 16:08:25 2019 +0000
@@ -29,7 +29,7 @@
   httpserver.start(-1);
   var channel = setupChannel(testpath);
   // ChannelListener defined in head_channels.js
-  channel.asyncOpen2(new ChannelListener(checkRequest, channel));
+  channel.asyncOpen(new ChannelListener(checkRequest, channel));
   do_test_pending();
   if (dbg) { print("============== test_plainText: out"); }
 });
@@ -40,7 +40,7 @@
   httpserver.start(-1);
   var channel = setupChannel(testpathGZip);
   // ChannelListener defined in head_channels.js
-  channel.asyncOpen2(new ChannelListener(checkRequest, channel,
+  channel.asyncOpen(new ChannelListener(checkRequest, channel,
                                         CL_EXPECT_GZIP));
   do_test_pending();
   if (dbg) { print("============== test_GZip: out"); }